wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubPersonalAccessTokenFetcher.java

```diff
@@ -205,21 +205,28 @@ public Optional<Boolean> isValid(PersonalAccessToken personalAccessToken) {
 
   @Override
   public Optional<Pair<Boolean, String>> isValid(PersonalAccessTokenParams params) {
-    if (!githubApiClient.isConnected(params.getScmProviderUrl())) {
-      LOG.debug("not a valid url {} for current fetcher ", params.getScmProviderUrl());
-      return Optional.empty();
+    GithubApiClient apiClient;
+    if (githubApiClient.isConnected(params.getScmProviderUrl())) {
+      apiClient = githubApiClient;
```

Comment:
I would add some comments here explaining that it is OAuth case.

Reply:
done